README.md

```diff
@@ -38,223 +38,42 @@ type stub packages (if found on PyPI).
-
-### pyright\_test.py
-
-This test requires Node.js to be installed. It is currently not part of the CI,
-but it uses the same pyright version and configuration as the CI.
-```
-(.venv3)$ python3 tests/pyright_test.py                 # Check all files
-(.venv3)$ python3 tests/pyright_test.py stdlib/sys.pyi  # Check one file
-```
-
-### mypy\_test\_suite.py
-
-This test requires Python 3.5 or higher; Python 3.6.1 or higher is recommended.
-Run using:
-```
-(.venv3)$ python3 tests/mypy_test_suite.py
-```
-
-This test runs mypy's own test suite using the typeshed code in your repo. This
-will sometimes catch issues with incorrectly typed stubs, but is much slower
-than the other tests.
-
-### check_consistent.py
-
-Run using:
-```
-python3 tests/check_consistent.py
-```
-
-### stubtest_test.py
-
-This test requires Python 3.6 or higher.
-Run using
-```
-(.venv3)$ python3 tests/stubtest_test.py
-```
-
-This test compares the stdlib stubs against the objects at runtime. Because of
-this, the output depends on which version of Python and on what kind of system
-it is run.
-Thus the easiest way to run this test is via Github Actions on your fork;
-if you run it locally, it'll likely complain about system-specific
-differences (in e.g, `socket`) that the type system cannot capture.
-If you need a specific version of Python to repro a CI failure,
-[pyenv](https://github.com/pyenv/pyenv) can help.
-
-Due to its dynamic nature, you may run into false positives. In this case, you
-can add to the whitelists for each affected Python version in
-`tests/stubtest_whitelists`. Please file issues for stubtest false positives
-at [mypy](https://github.com/python/mypy/issues).
-
-To run stubtest against third party stubs, it's easiest to use stubtest
-directly, with
-```
-(.venv3)$ python3 -m mypy.stubtest \
-  --custom-typeshed-dir <path-to-typeshed> \
-  <third-party-module>
-```
-stubtest can also help you find things missing from the stubs.
-
-
-### flake8
-
-flake8 requires Python 3.6 or higher. Run using:
-```
-(.venv3)$ flake8
-```
-
-Note typeshed uses the `flake8-pyi` and `flake8-bugbear` plugins.
+## Discussion
+
+If you've run into behavior in the type checker that suggests the type
+stubs for a given library are incorrect or incomplete,
+we want to hear from you!
+
+Our main forum for discussion is the project's [GitHub issue
+tracker](https://github.com/python/typeshed/issues).  This is the right
+place to start a discussion of any of the above or most any other
+topic concerning the project.
+
+For less formal discussion, try the typing chat room on
+[gitter.im](https://gitter.im/python/typing).  Some typeshed maintainers
+are almost always present; feel free to find us there and we're happy
+to chat.  Substantive technical discussion will be directed to the
+issue tracker.
+
+## Issue-tracker conventions
+
+We aim to reply to all new issues promptly.  We'll assign one or more
+labels to indicate we've triaged an issue, but most typeshed issues
```

Comment:
This seems to say "we'll assign one or more labels", but "we won't ad noise to the tracker by labelling all of [the issues]". Overall this section isn't saying much.

Reply:
And ... it's gone. I've removed the whole issue-tracker conventions section. Even the `[WIP]` label is outdated now that GitHub has draft PRs.